tempesta_fw/str.h

```diff
@@ -130,8 +135,8 @@ typedef struct {
 #define TFW_STR_DUP(s)		((s)->flags & TFW_STR_DUPLICATE)
```

Comment:
Comment https://github.com/tempesta-tech/tempesta/issues/215#issuecomment-221553926 about `TFW_STR_EMPTY()` was not addressed.